controllers/util/mcv_util.go

```diff
@@ -296,30 +304,41 @@ Requires:
 
 Returns: ManagedClusterView, error
 */
-func (m ManagedClusterViewGetterImpl) getOrCreateManagedClusterView(
+func (m ManagedClusterViewGetterImpl) getOrApplyManagedClusterView(
```

Comment:
Apply means create or update. We could name it getOrCreateOrUpdate but this is too horrible.

The real fix is to separate get from creation of the resource, but I don't want to rewrite this code now, this is just the minimal change to update existing mcvs.

Reply:
I think the function name should reflect what the function is doing, but since we don't have a good replacement I keep the exiting name. The real fix is to refactor this later so we don't mix 3 different operations in the same function.